.deps/prod.md

```diff
@@ -300,7 +301,7 @@
 | [`underscore@1.12.0`](git://github.com/jashkenas/underscore.git) | MIT | clearlydefined |
 | [`uri-js@4.2.2`](http://github.com/garycourt/uri-js) | BSD-2-Clause | [CQ18717](https://dev.eclipse.org/ipzilla/show_bug.cgi?id=18717) |
 | [`url-parse@1.4.7`](https://github.com/unshiftio/url-parse.git) | MIT | clearlydefined |
-| [`uuid@3.4.0`](https://github.com/uuidjs/uuid.git) | MIT | clearlydefined |
+| [`uuid@7.0.3`](https://github.com/uuidjs/uuid.git) | MIT |  |
```

Comment:
that's a bit strange that 3.4.0 is moved to dev dependencies and prod got 7.0.3 without clearlydefined

Reply:
I have fixed it as '3.4.0' in resolutions.